tests/test_upload.py

```diff
@@ -573,117 +391,12 @@ def mock_api_call(self, operation_name, api_params):
-            return None
-        raise NotImplementedError(key)
-
-    mock_bucket.get_blob = mock_get_blob
-
-    files_uploaded = []
-
-    def mocked_create_blob(key):
-        blob = gcsmock.MockBlob(key)
-
-        def mock_upload_from_file(file):
-            files_uploaded.append(key)
-
-        blob.upload_from_file = mock_upload_from_file
-        if key == "prefix/v0/xpcshell.dbg/A7D6F1BB18CD4CB48/xpcshell.sym":
-            pass
-        elif key == "prefix/v0/flag/deadbeef/flag.jpeg":
-            pass
-        else:
-            raise NotImplementedError(key)
-        return blob
-
-    mock_bucket.blob = mocked_create_blob
-
-    with open(ZIP_FILE, "rb") as f:
-        response = client.post(url, {"file.zip": f}, HTTP_AUTH_TOKEN=token.key)
-        assert response.status_code == 201
-
-        upload, = Upload.objects.all()
-        assert upload.user == fakeuser
-        assert upload.filename == "file.zip"
-        assert upload.completed_at
-        # Based on `ls -l tests/sample.zip` knowledge
-        assert upload.size == 70398
-        # This is predictable and shouldn't change unless the fixture
-        # file used changes.
-        assert upload.content_hash == "984270ef458d9d1e27e8d844ad52a9"
-        assert upload.bucket_name == "try"
-        assert upload.bucket_region is None
-        assert upload.bucket_endpoint_url == settings.UPLOAD_TRY_SYMBOLS_URL
-        assert upload.skipped_keys is None
-        assert upload.ignored_keys == ["build-symbols.txt"]
-        assert upload.try_symbols
-
-    assert "prefix/v0/xpcshell.dbg/A7D6F1BB18CD4CB48/xpcshell.sym" in files_uploaded
-    assert "prefix/v0/flag/deadbeef/flag.jpeg" in files_uploaded
-
-    assert len(upload_mock_update_uploads_created_task.all_delay_arguments) == 1
-
-    assert FileUpload.objects.all().count() == 2
-    file_upload = FileUpload.objects.get(
-        upload=upload,
-        bucket_name="try",
-        key="prefix/v0/flag/deadbeef/flag.jpeg",
-        compressed=False,
-        update=True,
-        size=69183,  # based on `unzip -l tests/sample.zip` knowledge
-    )
-    assert file_upload.completed_at
-
-    file_upload = FileUpload.objects.get(
-        upload=upload,
-        bucket_name="try",
-        key="prefix/v0/xpcshell.dbg/A7D6F1BB18CD4CB48/xpcshell.sym",
-        compressed=True,
-        update=False,
-        # Based on `unzip -l tests/sample.zip` knowledge, but note that
-        # it's been compressed.
-        size__lt=1156,
-        completed_at__isnull=False,
-    )
-
-
-@pytest.mark.django_db
-def test_upload_try_symbols_happy_path_s3(
     client,
     botomock,
     fakeuser,
-    metricsmock,
     upload_mock_invalidate_symbolicate_cache,
     upload_mock_update_uploads_created_task,
-    settings,
 ):
-    settings.UPLOAD_DEFAULT_URL = "https://s3.example.com/private/prefix/"
-    settings.UPLOAD_TRY_SYMBOLS_URL = "https://s3.example.com/try/prefix"
```

Comment:
When GCS was the default for tests, it was necessary to set these settings for S3 test variants. With S3 as the default again, it is not needed, and in some cases, the settings fixture is not needed either.